Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/MigrationCheck.kt

```diff
@@ -0,0 +1,33 @@
+package de.rki.coronawarnapp.covidcertificate.person.core
+
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CertificatePersonIdentifier
+import de.rki.coronawarnapp.main.CWASettings
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class MigrationCheck @Inject constructor(private val cwaSettings: CWASettings) {
+
+    fun shouldShowMigrationInfo(persons: Set<PersonCertificates>): Boolean {
+        if (cwaSettings.wasCertificateGroupingMigrationAcknowledged) return false
+        cwaSettings.wasCertificateGroupingMigrationAcknowledged = true
+
+        val numberOfPersons = persons.size
+
+        val numberOfPersonIdentifiers = persons
```

Comment:
Maybe call it `numberOfPersonsWithLegacyGrouping` to increase understandability?

Reply:
ok, it will be updated in the another PR